deploy/catalog_resources/community/1.2.1/kiecloud-operator.1.2.1.clusterserviceversion.yaml

```diff
@@ -1,216 +1,228 @@
-            displayName: Business/Decision Central URL
-            path: consoleHost
-            x-descriptors:
-              - "urn:alm:descriptor:org.w3:link"
-  install:
-    strategy: deployment
-    spec:
-      permissions:
-        - serviceAccountName: kie-cloud-operator
-          rules:
-            - apiGroups:
-                - ""
-                - apps
-                - app.kiegroup.org
-                - rbac.authorization.k8s.io
-                - apps.openshift.io
-                - image.openshift.io
-                - build.openshift.io
-                - route.openshift.io
-              resources:
-                - "*"
-              verbs:
-                - "*"
-            - apiGroups:
-                - monitoring.coreos.com
-              resources:
-                - servicemonitors
-              verbs:
-                - get
-                - create
-            - apiGroups:
-                - operators.coreos.com
-              resources:
-                - clusterserviceversions
-              verbs:
-                - "*"
-            - apiGroups:
-                - apps
-              resourceNames:
-                - kie-cloud-operator
-                - business-automation-operator
-              resources:
-                - deployments/finalizers
-              verbs:
-                - update
-      deployments:
-        - name: kie-cloud-operator
-          spec:
-            replicas: 1
-            strategy:
-              type: RollingUpdate
-              rollingUpdate:
-                maxSurge: 1
-                maxUnavailable: 1
-            selector:
-              matchLabels:
-                name: kie-cloud-operator
-            template:
-              metadata:
-                labels:
-                  name: kie-cloud-operator
-              spec:
-                serviceAccountName: kie-cloud-operator
-                containers:
-                  - name: kie-cloud-operator
-                    image: quay.io/kiegroup/kie-cloud-operator:1.2.1
-                    ports:
-                      - containerPort: 60000
-                        name: metrics
-                    command:
-                      - kie-cloud-operator
-                    imagePullPolicy: Always
-                    env:
-                      - name: WATCH_NAMESPACE
-                        valueFrom:
-                          fieldRef:
-                            fieldPath: metadata.namespace
-                      - name: POD_NAME
-                        valueFrom:
-                          fieldRef:
-                            fieldPath: metadata.name
-                      - name: OPERATOR_NAME
-                        valueFrom:
-                          fieldRef:
-                            fieldPath: metadata.labels['name']
-                      - name: OPERATOR_UI
-                        value: "true"
+  version: 1.2.1
```

Comment:
Why the changes? Are you committing generated versions?

Reply:
correct, this is the generated version and should only have this many changes this first time. For the most part, though, it's just a reordering of the fields.